beacon_node/http_api/src/lib.rs

```diff
@@ -3864,6 +3905,7 @@ pub fn serve<T: BeaconChainTypes>(
                     .uor(post_validator_prepare_beacon_proposer)
                     .uor(post_validator_register_validator)
                     .uor(post_lighthouse_liveness)
+                    .uor(post_lighthouse_liveness_epoch)
```

Comment:
this should be renamed `post_validator_liveness` and moved up with the other `post_validator_*` methods